src/flowr/internal-session.ts

```diff
@@ -24,12 +21,10 @@ export class FlowrInternalSession {
 		void this.shell.usedRVersion().then(version => {
 			this.outputChannel.appendLine(`Using R shell: ${JSON.stringify(version)}`)
 		})
-		process.on('exit', () => {
```

Comment:
In a way we should still have them, but link to destroy. we want a clean exit (even though, probably, the node runtime would cover us here. we want this to stay explicit).

Reply:
But, theoretically, we can move this to the extension

Reply:
(also do this on `SIGINT` is what we want)